src/dispensing-tiles/dispensing-tiles.component.tsx

```diff
@@ -0,0 +1,42 @@
+import React from "react";
+import { useTranslation } from "react-i18next";
+import { DataTableSkeleton } from "carbon-components-react";
+import { useMetrics } from "./dispensing-tiles.resource";
+import DispensingTile from "./dispensing-tile.component";
+import styles from "./dispensing-tiles.scss";
+
+const DispensingTiles: React.FC = () => {
+  const { t } = useTranslation();
+  const { metrics, isError, isLoading } = useMetrics();
+
+  if (isLoading) {
+    return <DataTableSkeleton role="progressbar" />;
+  }
+
+  return (
+    <>
+      <div className={styles.cardContainer}>
+        <DispensingTile
+          label={t("orders", "Orders")}
+          value={metrics ? metrics.orders : 0}
```

Comment:
```suggestion
          value={metrics.orders}
```

This shouldn't show `0` when in an error or not loaded state. `0` is information, and it would be wrong in that case.